src/components/_list-view.scss

```
@@ -669,6 +728,12 @@ $list-view-gaps: $gaps;
   display: flex;
   justify-content: center;
   align-items: center;
+  &.-arrow {
```

[IMO] ここは形状に着目するより、`toggle` みたいな機能に着目した名前のほうが良いかも。

```suggestion
  &.-toggle {
```